pymc/tests/test_distributions_timeseries.py

```diff
@@ -200,13 +187,13 @@ def test_inconsistent_steps_and_shape(self):
     def test_inferred_steps_from_dims(self):
         with pm.Model(coords={"batch": range(5), "steps": range(20)}):
             x = GaussianRandomWalk("x", dims=("batch", "steps"))
-        steps = x.owner.inputs[-1]
+        steps = x.owner.inputs[0].owner.inputs[2].owner.inputs[1][-1]
```

Comment:
Getting the steps parameter this way still passes the test, but the dimension is one too big to my understanding (again, c.f. the classmethod `change_size`)